demo/mmediting_inference_tutorial.ipynb

```diff
@@ -0,0 +1,346 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# MMEditing Inference Tutorial\n",
+    "\n",
+    "Welcome to MMEditing! In this tutorial you will learn how to use MMEditing inference api to predict your own image or video. \n",
+    "\n",
+    "This is a quick guide for you to infer with existing models. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Install MMEditing\n",
+    "Please refer to [README.md](README.md) for installation instruction."
```

Comment:
users will expect to experience all the demos within this tutorial rather than jumping out to the readme for installation. Please provide the best practice of installation here. Refer to https://github.com/open-mmlab/mmocr/blob/dev-1.x/demo/MMOCR_Tutorial.ipynb

Reply:
good idea. installation instruction is provided. resolved